microservice/lpwan-custom-codec/readme.md

```diff
@@ -280,8 +200,12 @@ The fragments above are used as follows:
 
 ```json
 {
-    "encodedCommand": "9F5000",
-    "fport": 20
+  "encodedCommand": "9F000000",
+  "properties": {
+    "fport": "100"
+  },
+  "message": "Successfully Encoded the payload",
```

Comment:
not sure if this is something we can even change in the scope of this PR, but i think it should be "Encoded" -> "encoded" (lower case).

Reply:
@rssag this message is coming from the example microservice lora-codec-lansitec. Hence the case is coming in this manner. We could correct it in a subsequent PR.